submission/persistence/src/main/kotlin/ac/uk/ebi/biostd/persistence/common/SubmissionTypes.kt

```diff
@@ -1,8 +1,7 @@
 package ac.uk.ebi.biostd.persistence.common
 
-enum class SubmissionTypes(val value: String) {
+internal enum class SubmissionTypes(val value: String) {
     Study("Study"),
     Project("Project"),
     Array("Array"),
-    Compound("Compound");
```

Comment:
For consistency, we should probably keep Compound in or get rid of all unused submission types (everything but Project?)